render/mapping.go

```diff
@@ -67,34 +132,7 @@ func ProcessPID(m report.NodeMetadata) (RenderableNode, bool) {
 // processes with the same name together).
 func ProcessName(m report.NodeMetadata) (RenderableNode, bool) {
 	show := m["pid"] != "" && m["name"] != ""
-	return newRenderableNode(m["name"], m["name"], "", m["name"]), show
-}
-
-// MapEndpoint2Container maps endpoint topology nodes to the containers they run
-// in. We consider container and image IDs to be globally unique, and so don't
-// scope them further by e.g. host. If no container metadata is found, nodes are
-// grouped into the Uncontained node.
-func MapEndpoint2Container(m report.NodeMetadata) (RenderableNode, bool) {
-	var id, major, minor, rank string
-	if m["docker_container_id"] == "" {
-		id, major, minor, rank = "uncontained", "Uncontained", "", "uncontained"
-	} else {
-		id, major, minor, rank = m["docker_container_id"], "", m["domain"], ""
-	}
-
-	return newRenderableNode(id, major, minor, rank), true
-}
-
-// MapContainerIdentity maps container topology node to container mapped nodes.
-func MapContainerIdentity(m report.NodeMetadata) (RenderableNode, bool) {
-	var id, major, minor, rank string
-	if m["docker_container_id"] == "" {
-		id, major, minor, rank = "uncontained", "Uncontained", "", "uncontained"
-	} else {
-		id, major, minor, rank = m["docker_container_id"], m["docker_container_name"], m["domain"], m["docker_image_id"]
-	}
-
-	return newRenderableNode(id, major, minor, rank), true
+	return NewRenderableNode(m["name"], m["name"], "", m["name"], m), show
```

Comment:
I think it's time to issue a ban on all "name" keys in NodeMetadatas, and always qualify them. In this case I think it should be "host_name"—is that right?